spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/AuditActionType.java

```diff
@@ -33,7 +33,8 @@
 	DEPLOY(300L, "Deploy", "Deploy an Entity"),
 	ROLLBACK(400L, "Rollback", "Rollback an Entity"),
 	UNDEPLOY(500L, "Undeploy", "Undeploy an Entity"),
-	UPDATE(600L, "Update", "Update an Entity");
+	UPDATE(600L, "Update", "Update an Entity"),
+	LOGIN_SUCCESS(700L, "Login", "Successful login");
```

Comment:
Instead of `Login` we could have `SuccessfulLogin` as this action involves only the successful login.